frontend/app/src/utils/text.ts

```diff
@@ -12,6 +12,21 @@ export const toSentenceCase = (string: string): string => {
   return string[0].toUpperCase() + string.slice(1);
 };
 
+/**
+ *
+ * @param {string} string - String to convert
+ * @return {string} - String converted to text token, mostly used to matching keyword
+ * @example
+ * toSnakeCase('this is a sentence'); // thisisasentence
```

Comment:
I guess this in the documentation is a typo right? the wrong method name

Reply:
yes, will update it and merge